package.json

```diff
@@ -118,6 +118,7 @@
     "serialize-error": "^1.1.0",
     "set-immediate-shim": "^1.0.1",
     "source-map-support": "^0.4.0",
+    "stack-utils": "github:tapjs/stack-utils#long-stack-traces",
```

Comment:
I just did

```
npm i -S tapjs/stack-utils#long-stack-traces
```

and that is how it modified `package.json`. I want to get that PR merged before we merge this anyways.